lib/mayaUsd/commands/layerEditorCommand.cpp

```diff
@@ -620,7 +621,7 @@ class MuteLayer : public BaseCmd
         // we perfer not holding to pointers needlessly, but we need to hold on to the layer if we
         // mute it otherwise usd will let go of it and its modifications, and any dirty children
         // will also be lost
```

Comment:
I dont think we need this comment anymore

Reply:
Actually, the comment says exactly why we are calling addMutedLayer, so I'll keep the comment.